pages/doc/logging_send_logs.md

```diff
@@ -62,15 +59,23 @@ To install and configure a new proxy:
 
 {% include note.html content="To learn more about the proxy configuration properties and preprocessor rules for logs, see [Logs Proxy Configurations and Preprocessor Rules](logging_proxy_configurations.html)." %}
 
-## Configure the Log Shipper
+## Option 1: Use Our Integrations
+
+You can monitor your Kubernetes clusters or Linux hosts using our built-in integrations and send logs to our system.
+
+* [Linux host integration](linux.html): Install the Wavefront proxy and configure the log shipper.
+* [Kubernetes integration](kubernetes.html#kubernetes-quick-install-using-the-kubernetes-operator): Enable logs in the integration, generate the script, and run it on your Kubernetes cluster. 
```

Comment:
Enable logs while you set up the integration, generate the script, ...

Maybe also add Amazon Web Services? We released the AWS Logs Beta Setup last week. I have included the steps here: https://docs.wavefront.com/integrations_aws_metrics.html#setup-for-ingesting-aws-cloudwatch-logs-beta.

Reply:
Hi Ita,
Thank you! I will update this section with another PR.